WordPress/src/main/java/org/wordpress/android/ui/plans/PlansUtils.java

```diff
@@ -95,21 +95,41 @@ public static String getIconUrlForPlan(Plan plan, int iconSize) {
     }
 
     public static boolean isFreePlan(Plan plan) {
-        return plan.getProductSlug().equals("free_plan") ||
-               plan.getProductSlug().equals("jetpack_free");
+        return plan.getProductSlug().equals(PlansConstants.FREE_PLAN_SLUG) ||
+               plan.getProductSlug().equals(PlansConstants.JETPACK_FREE_PLAN_SLUG);
     }
 
     /**
      * Weather the plan ID is a free plan.
      *
-     * 2002 = Jetpack free
-     * 1    = WordPress.com free
-     *
      * @param planID - The plan ID
      * @return boolean - true if the current blog is on a free plan.
      */
     public static boolean isFreePlan(long planID) {
-        return planID == 2002 || planID == 1;
+        return planID == PlansConstants.JETPACK_FREE_PLAN_ID || PlansConstants.FREE_PLAN_ID == 1;
```

Comment:
Instead of `PlansConstants.FREE_PLAN_ID == 1` this should be `planID == PlansConstants.FREE_PLAN_ID`